netty/src/main/java/io/grpc/netty/NettyServerTransport.java

```diff
@@ -58,6 +59,9 @@
   private final int flowControlWindow;
   private final int maxMessageSize;
   private final int maxHeaderListSize;
+  // TODO(zdapeng): allow custom keep alive config values by NettyServerBuilder
+  private final long keepAliveTimeInNanos = TimeUnit.NANOSECONDS.convert(2L, TimeUnit.HOURS);
```

Comment:
Can you default to infinite for this PR, so the feature is disabled? Once configuration is available, then we can turn it on (so if there are problems, there is a way to disable it).